CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 * Using flexible dependacies instead of locked versions
 * Moved sizecheck to a separate dev dependacy
 * Added web3Provider Support [#57]
+* Added factories Resolution.infura and Resolution.provider
```

Comment:
Please add transition methods for web3, ethereumjs and ethers.

Reply:
All of them are using either .send or .sendAsync internally. I added the jsonRPCprovider which supposed to cover web3, ethereumjs and ethers providers as well as any other with .send function instead of .sendAsync